jwql/database/database_interface.py

```diff
@@ -365,10 +380,26 @@ class : obj
 
     return type(class_name, (base,), data_dict)
 
+
 # Create tables from ORM factory
-# NIRCamDarkQueries = monitor_orm_factory('nircam_dark_queries')
-# NIRCamDarkPixelStats = monitor_orm_factory('nircam_dark_pixel_stats')
-# NIRCamDarkDarkCurrent = monitor_orm_factory('nircam_dark_dark_current')
+definitions_dir = os.path.join(os.path.split(__file__)[0], 'monitor_table_definitions')
```

Comment:
@bhilbert4 I don't see where/how this `definitions_dir` variable is used.  Could it be removed?

Reply:
Ah yes. That's leftover from when I didn't realize that `get_monitor_columns` contains the assumed location of the text files.